forge-gui/res/cardsfolder/upcoming/iron_craw_crusher.txt

```diff
@@ -0,0 +1,11 @@
+Name:Iron-Craw Crusher
+ManaCost:7
+Types:Artifact Creature Wurm
+PT:4/6
+K:Prototype:2 G G:2:5
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME Crusher attacks, target attacking creature gets +X/+0 until end of turn, where X is CARDNAME's power.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature you control | NumAtt$ X
```

Comment:
no "you control" unless the oracle here is wrong? Either way something needs to be fixed